code/game/machinery/atmoalter/canister.dm

```diff
@@ -193,6 +194,7 @@ update_flag
 
 /obj/machinery/portable_atmospherics/canister/process()
 	src.updateDialog()
```

Comment:
Oops, tgui handles all of this nonsense. Please remove all calls to updateDialog/updateUsrDialog.